src/language/providers/elementCompletion.ts

```diff
@@ -18,8 +18,9 @@
 import * as vscode from 'vscode'
 import { checkMissingCloseTag, getCloseTag } from './closeUtils'
 import {
+  _XmlItem,
```

Comment:
Are you sure we're supposed to be using this `_XmlItem` directly?

Reply:
I'm not sure how to answer, what would you suggest?